src/OpenTelemetry.Instrumentation.Process/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 0.5.0
```

Comment:
This version can be confusing. I do not think that it is good idea to release 0.5.0 after 1.0 (alpha)

Reply:
The spec is not stable yet and based on the following statement, I think 0.X.0 without any identifier may be a good fit.
https://semver.org/#spec-item-4

But I do see your concern.
Would 1.0.0-prerelease.1 makes more sense?

Reply:
there is proposal, even in the main repo to release the instrumentation libraries as stable (i.e without prerelease identifiers), however with < 1.0.0 as version.

0.5.0 is doable, but lets first do 0.5.0-beta/rc and wait for weeks.

the 1.0.0-alpha out there could be delisted to avoid confusion?

Reply:
Delisting seems to be good compromise.
Why starting from some random value? Why not 0.1.0?
What should be order `1.0.0-prerelease.1`? alpha, beta, prerelease, then rc (I suppose that nuget will treat it alphabetically)?

IMO, delisting + start from some low number looks like the best option.

@cijothomas, it will be great to cleanup a bit also old contrib packages: https://www.nuget.org/packages?q=OpenTelemetry.Contrib and redirect them to version without `Contrib` infix. If not possible, also delist them.

Reply:
> Why starting from some random value? Why not 0.1.0?

TBH, i do not have a preference if it is 0.1.0 or 0.5.0 as long as it is < 1.0.0.

We are planning to try this for instrumentaion library in main repo. This package is a good way to test the process. (as we will need to delist 1.* from nuget here too.)